api/server/router/image/image_routes.go

```diff
@@ -222,7 +222,16 @@ func (s *imageRouter) getImagesGet(ctx context.Context, w http.ResponseWriter, r
 		names = r.Form["names"]
 	}
 
-	if err := s.backend.ExportImage(names, output); err != nil {
+	format := r.FormValue("format")
+	refs := map[string]string{}
+	refsJSON := r.FormValue("refs")
```

Comment:
The new options should be ignored on older API versions, and a default should be set (if needed) for older API versions

Reply:
probably need to ignore new options then

Reply:
the default is already set